src/main/resources/default/assets/tycho/scripts/focus.js

```diff
@@ -0,0 +1,44 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+sirius.ready(function() {
+
+    function focus(field) {
+        field.focus();
```

Comment:
Optional: use the _ prefix for all DOM elements in this file, such as _field, _primaryField, _form, ...